src/test/components/TooltipCallnode.test.js

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import React from 'react';
+import { Provider } from 'react-redux';
+import { TooltipCallNode } from '../../components/tooltip/CallNode';
+import { render } from 'react-testing-library';
+import { storeWithProfile } from '../fixtures/stores';
+import { getProfileWithNativeAllocations } from '../fixtures/profiles/processed-profile';
+import { selectedThreadSelectors } from '../../selectors/per-thread';
+import * as ProfileSelectors from '../../selectors/profile';
+import * as UrlState from '../../selectors/url-state';
+import {
+  changeSelectedCallNode,
+  changeCallTreeSummaryStrategy,
+} from '../../actions/profile-view';
+import { ensureExists } from '../../utils/flow';
+
+describe('TooltipCallNode', function() {
+  function setup(profile) {
+    const store = storeWithProfile(profile);
+    const { getState, dispatch } = store;
+
+    function renderTooltip(): * {
+      // This component is not currently connected.
+      return render(
+        <Provider store={store}>
```

Comment:
Why do you use the Provider if the tested component isn't connected ?

Reply:
I copied/pasted it in. I think I'll leave it in case in the future someone adds a child component that is connected. It's not hurting anything as is, and is close the environment in which we actually use it.